.github/workflows/rerun-workflows.yml

```diff
@@ -19,17 +19,78 @@ jobs:
           echo "View the failed run attempt (#${{ github.event.workflow_run.run_attempt }}) using the following link:" >> $GITHUB_STEP_SUMMARY
           echo "${{ github.event.workflow_run.html_url }}" >> $GITHUB_STEP_SUMMARY
 
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - run: npm install @slack/web-api
```

Comment:
this pattern is recommended in the github actions scripts docs
https://github.com/actions/github-script#use-npm-packages